ui/app/AppLayouts/Communities/CommunitiesPortalLayout.qml

```diff
@@ -137,14 +137,18 @@ StatusSectionLayout {
                     onClicked: Global.importCommunityPopupRequested()
                 }
 
-                StatusButton {
-                    id: createBtn
-                    objectName: "createCommunityButton"
-                    Layout.preferredHeight: 38
-                    verticalPadding: 0
-                    text: qsTr("Import existing community from Discord")
-                    onClicked: {
-                        Global.createCommunityPopupRequested(true /*isDiscordImport*/)
+                Loader {
+                    Layout.preferredHeight: active ? 38 : 0
+                    active: communitiesStore.createCommunityEnabled || communitiesStore.testEnvironment
+                    sourceComponent: StatusButton {
+                        id: createBtn
+                        objectName: "createCommunityButton"
```

Comment:
This won't be visible by tests now since it's inside the Loader

Reply:
I set the Loader active when `testEnvironment` is `true`. Do I need to do something else?